### PR TITLE
Add population size plot

### PR DESCRIPTION
**Pull request by svenpopping**

### CODE REVIEW

File: plots/plot_gp_pop_size.py [hunk 1]
```diff
@@ -0,0 +1,57 @@
...
+
+def extract_mean_std(rest):
+    split = rest.split(' ')
+    return round(float(split[1]), 2), round(float(split[2]), 2)
+
+
+def extract_value_per_run(size):
+    fileHandle = open(f"../log/log_pop_size_{size}.txt", "r")
+    lineList = fileHandle.readlines()
+    fileHandle.close()
```
Comment (kwendel) [inline plots/plot_gp_pop_size.py]: with open("bla") as f:
    lineList = f.readLines()

File: plots/plot_gp_pop_size.py [hunk 2]
```diff
@@ -0,0 +1,57 @@
...
+
+    results = lineList[len(lineList) - 4:]
```
Comment (kwendel) [inline plots/plot_gp_pop_size.py]: results = lineList[-4]

File: test_tuner_pop_size.py [hunk 3]
```diff
@@ -0,0 +1,38 @@
...
+from simplegp.Nodes.SymbolicRegressionNodes import *
+from simplegp.Weights.Tuner import Tuner
+
+np.random.seed(42)
+random.seed(42)
+
+settings = [10, 20, 50, 100, 500, 1000]
+
+
+def run_with_range(range):
```
Comment (m9tdev) [inline test_tuner_pop_size.py:20]: `range`?????

File: test_tuner_pop_size.py [hunk 4]
```diff
@@ -0,0 +1,38 @@
...
+    tuner = Tuner(
+        pop_size=range,
+        run_generations=(99, 100)
+    )
+    sgp = SimpleGP(tuner=tuner, functions=functions, pop_size=100, max_generations=100)
+
+    CrossValidation(sgp, terminals).validate()
+
+
+for setting in tqdm(settings, desc="Test Weights"):
```
Comment (kwendel) [inline test_tuner_pop_size.py]: Does not work on my machine

**Review by kwendel — CHANGES_REQUESTED** [review]

File: test_tuner_pop_size.py (hunk 4 above)
Comment (svenpopping) [reply test_tuner_pop_size.py]: we dont support windows, vies!

**Review by timovanasten — CHANGES_REQUESTED** [review]
Please remove 'timo mag niet meedoen' label

**Pull request merged**